Cargo.toml

```diff
@@ -4,16 +4,19 @@ version = "0.9.1"
 edition = "2021"
 
 [dependencies]
+endpoints = { git = "https://github.com/suryyyansh/LlamaEdge.git", branch = "dev"}
+llama-core = { git = "https://github.com/suryyyansh/LlamaEdge.git", branch = "dev", features = ["full"]}
+chat-prompts = { git = "https://github.com/suryyyansh/LlamaEdge.git", branch = "dev"}
```

Comment:
Please use `endpoints v0.13.1`, `chat-prompts v0.13.0`, and `llama-core v0.16.1`. Thanks!